app/controllers/application_pets_controller.rb

```diff
@@ -11,4 +11,13 @@ def update
     redirect_to "/pets/#{pet.id}"
   end
 
+   def batch_update 
+    pet_ids = params[:pet_ids]
```

Comment:
Update each pet id individually using .each to set status to 'pending' on each pet's show page